rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/robolectric/activity/CovidJsonFormActivityTest.java

```diff
@@ -37,7 +37,7 @@
     @Before
     public void setUp() throws Exception {
         super.setUp();
-        jsonFormActivity = Robolectric.buildActivity(CovidJsonFormActivity.class, intent).create().resume().get();
+        jsonFormActivity = Robolectric.buildActivity(CovidJsonFormActivity.class, intent).create().get();
```

Comment:
Any reason for this change?

Reply:
https://github.com/onaio/rdt-standard/pull/945#discussion_r602226539

Reply:
could this be addressed as well?

Reply:
done.

Reply:
this is still not addressed, revert the changes here.

Reply:
and verify that the `Utils.verifyAuthorization` is invoked for all activities in which it's added.

Reply:
done

Reply:
changes reverted